src/Tasks/Microsoft.CSharp.CurrentVersion.targets

```diff
@@ -84,7 +84,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     -->
 
     <PropertyGroup>
-        <CreateManifestResourceNamesDependsOn></CreateManifestResourceNamesDependsOn>
+        <CreateManifestResourceNamesDependsOn>$(CreateManifestResourceNamesDependsOn)</CreateManifestResourceNamesDependsOn>
```

Comment:
While I wasn't able to find any incidence of this being overriden by users, the users might possibly be doing it in non-public codes.

If someone would override (or is overriding) the previously suggested way (explicit import of sdk.props and sdk.targets and then chaining it after the sdk.targets imports) - we'd be breaking that usage

Reply:
How would deleting this line be a break? Before it explicitly unset the property, now it sets it to current value, I'm saying don't set it to anything.

```suggestion
```

Reply:
Oh - you mean delete just the definition line, but keep using it in `DependsOnTargets` of `CreateManifestResourceNames` target?

That would be fine. Just probably going against the practice we are now trying to push - of not trying to use undefined properties.


If you meant removing it even from the `DependsOnTargets`, than it would break case where someone defines that property after explicitly importing sdk.targets.

Reply:
Ah: I meant just this line (and the others that don't do anything).

IMO we should define properties in some different way, or maybe have a pattern-based exception for `*DependsOn` properties used in `DependsOnTargets` or something. Every line of XML has a (small!) cost and I'd rather tend to mimimalism.

Reply:
Makes sense - let's do that (both - deleting and special casing DependsOn in undefined read checks)

Reply:
I went all through those file and removed all no-op DependsOn